lib/dialects/postgres/query-generator.js

```diff
@@ -101,58 +101,128 @@ const QueryGenerator = {
     if (!schema) {
       schema = 'public';
     }
-    return 'SELECT pk.constraint_type as "Constraint", c.column_name as "Field", ' + 
```

Comment:
@sushantdhiman I reverted the indentation, but trailing whitespace has been removed since `.editorconfig` overrides ``trim_trailing_whitespace = true``.

If you want revert the whitespaces as well, I'll try different editor.